src/components/molecules/LiveSchedule/LiveSchedule.tsx

```diff
@@ -0,0 +1,86 @@
+import { useSelector } from "hooks/useSelector";
+import { useUser } from "hooks/useUser";
+import React, { FC } from "react";
+import { VenueEvent } from "types/VenueEvent";
+import { isEventLive } from "utils/event";
+import { currentTimeInUnixEpoch, formatHourAndMinute } from "utils/time";
+import { isExternalUrl } from "utils/url";
+import { enterRoom } from "utils/useLocationUpdateEffect";
+import "./LiveSchedule.scss";
+
+const LiveSchedule: FC = () => {
+  const { user, profile } = useUser();
+  const { venueEvents, venue } = useSelector((state) => ({
+    venueEvents: state.firestore.ordered.venueEvents,
+    venue: state.firestore.ordered.currentVenue[0],
+  }));
+
+  const liveEvents =
+    venueEvents && venueEvents.length
+      ? venueEvents.filter((event) => isEventLive(event))
+      : [];
```

Comment:
As above RE: `useMemo`, extracting `filter` functions (though in this case it's so simple that it probably doesn't make a difference so long as we memo the actual filtering/output)

Reply:
fixed in 77fc5d6d1829a42da2152d00871cde0b6f47269d